### PR TITLE
Iss9

### PR DESCRIPTION
**Pull request by LPMendoza**
## Reports added
In this issue were added the reports: 
* Attendeces by day
* Absences by month
* Average of check in/out by month

image
image

## To test it.
After download the project, you will have to setup the database configuration:

In the file /config/database.yml you have to type the credential of you database.
Then type in the terminal the next command: **rails db:create db:migrate**
This command will create the database and tables of the project.
Also you have to run: **rake db:seed**, to create the default admin user (admin@example.com, password)
Then in the left menu you can access and try the reports.

### CODE REVIEW

**Comment by MiguelOcegueraM** [comment]
Everythings looks good! go ahead with the next step!

**Review by MiguelOcegueraM — APPROVED** [review]
Everythings looks good! go ahead with the next step!

**Review by n0b0dy-su — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -33,6 +31,10 @@ gem 'bootsnap', '>= 1.4.2', require: false
 # Bootstrap sass
 gem 'bootstrap-sass'
 gem "bootstrap_form", "~> 4.0"
+gem 'devise'
+gem 'activeadmin'
+gem 'arctic_admin'
```
Comment (OchoaHub) [inline Gemfile:36]: This theme looks good;)

File: app/admin/absences.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register_page 'Absences' do
+  menu parent: 'Reports'
```
Comment (OchoaHub) [inline app/admin/absences.rb:4]: I like the idea with tab menu +10

**Review by glorianayeli — APPROVED** [review]
good work, I can see that you implement the abstraction and encapsulation in the correct way.

**Review by OchoaHub — APPROVED** [review]
I think it's fine

**Comment by EduardoGHdez** [comment]
I'd only suggest rebasing your branch with latest master before merging or asking for a review.  This in order to keep the git-history cleaner and easier to walk through 🚶🏽 .

I can see you have a couple of merge-commits that might be coming from previous merges into the project.

If you are interested in the workflow I'm suggesting, try reading this resource 😄 : https://www.atlassian.com/git/tutorials/merging-vs-rebasing

Another thing that I noticed is that your git-messages can improve 👍🏽 
Take a look at this another classic resource that talks about the importance of writing more descriptive commit-messages and the formatting some people like to follow: https://chris.beams.io/posts/git-commit/

**Pull request merged**